app/views/projects/index/_project_actions_dropdown.html.erb

```diff
@@ -48,6 +48,17 @@
           <% end %>
         </li>
       <% end %>
+      <!-- Project members access -->
+      <% if can_read_project?(project) %>
+        <li class="form-dropdown-item">
+          <%= link_to can_manage_project_access?(project) ? edit_access_permissions_project_path(project, format: :json) : access_permissions_project_path(project, format: :json),
```

Comment:
doesn't it default to json?

Reply:
@okriuchykhin can you merge the master in you `biosistemika:features/new_permissions` branch? I'm not able to push directly